specificatie/naam.yaml

```diff
@@ -0,0 +1,145 @@
+openapi: 3.0.3
+info:
+  title: Naam definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    AbstractNaamBasis:
+      type: object
+      properties:
+        type:
+          type: string
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaNaam: '#/components/schemas/GbaNaamBasis'
+          Naam: '#/components/schemas/NaamBasis'
+    GbaNaamBasis:
+      type: object
+      properties:
+        voornamen:
+          type: string
+          maxLength: 200
+          description: |
+            De verzameling namen voor de geslachtsnaam, gescheiden door spaties.
+        voorvoegsel:
+          type: string
+          maxLength: 10
+          example: "de"
+        geslachtsnaam:
+          type: string
+          maxLength: 200
+          description: |
+            De achternaam van een persoon.
+          example: "Vries"
+        adellijkeTitelPredikaat:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+    NaamBasis:
+      allOf:
+        - $ref: '#/components/schemas/GbaNaamBasis'
+        - type: object
+          properties:
+            voorletters:
+              type: string
+              description: |
+                De voorletters van de persoon, afgeleid van de voornamen.
+              example: "P.J."
+    Naam:
```

Comment:
Mis ik hier nu de GbaNaam waar bij de GbaNaamBasis met GbaInOnderzoek wordt gedefinieerd of snap ik de opzet niet (dat laatste is nog steeds een reële optie hoor. )

Reply:
Voor zover ik gevonden heb wordt Naam nu nergens gebruikt

Reply:
Was de oude NaamGerelateerde. Door het toevoegen van VolledigeNaam aan NaamBasis moest een variant worden gedefinieerd zonder die property. Ik heb toen niet gekeken of de NaamType wordt verwezen. Wordt verwijderd